src/components/loader-spinner/loader-spinner.component.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import { MarginProps } from "styled-system";
+import tagComponent, {
+  TagProps,
+} from "../../__internal__/utils/helpers/tags/tags";
+import {
+  StyledSpinnerWrapper,
+  StyledLabel,
+  StyledSpinnerCircleSvg,
+} from "./loader-spinner.style";
+import useLocale from "../../hooks/__internal__/useLocale";
+import useMediaQuery from "../../hooks/useMediaQuery";
+import Typography from "../typography";
+
+export type LoaderSpinnerSizes = "XS" | "S" | "M" | "L" | "XL";
```

Comment:
comment: I see why you've gone with the short-hand here as we have a few components like it in the codebase. However, I think we should go with the long form values here as we've adopted that more often in other components including [loader](https://github.com/Sage/carbon/blob/ab116604a60104ca15241750558a211f4c326d0b/src/components/loader/loader-square.style.ts). I think we should raise a ticket to align the others that have this approach currently etc